Ginger/GingerCoreNET/ActionsLib/ActionsConversion/ActionConversionUtils.cs

```diff
@@ -65,105 +65,227 @@ public string ActUIElementClassName
             set;
         }
 
+        bool mStopConversion = false;
+
+        public ObservableList<BusinessFlowConversionStatus> ListOfBusinessFlow = new ObservableList<BusinessFlowConversionStatus>();
+
         /// <summary>
-        /// This method is used to add the actions
+        /// This method stops the multiple businessflow action conversion process
+        /// </summary>
+        public void StopConversion()
+        {
+            mStopConversion = true;
+        }
+
+        /// <summary>
+        /// This method stops the multiple businessflow action conversion process
+        /// </summary>
+        public void ContinueConversion(bool addNewActivity,
```

Comment:
we said 'addNewActivity' won't be first param, right? relevant to all below methods

Reply:
changed the order